ui/component/userVerify/view.jsx

```diff
@@ -120,25 +120,17 @@ class UserVerify extends React.PureComponent<Props> {
           </div>
 
           <Card
-            icon={ICONS.CHAT}
-            title={__('Verify via chat')}
+            icon={ICONS.HELP}
+            title={__('Verify via email')}
             subtitle={
               <>
                 <p>
-                  {__(
-                    'A moderator can approve you within our discord server. Please review the instructions within #rewards-approval carefully.'
-                  )}
+                  {__('You can request verification of your Odysee account by sending an email to help@odysee.com')}
                 </p>
-                <p>{__('You will be asked to provide proof of identity.')}</p>
+                <p>{__('Verification requests can take a few hours to be approved.')}</p>
```

Comment:
@michaelmitnick please add this to the app-strings.json

Reply:
meant "'Verification requests can take a few hours to be approved." string to app-strings.json. Let me know if you wanna take another shot.  

Notice you also added back: 'You will be asked to provide proof of identity.' - that's not what I meant, and probably can be removed?

Reply:
Oh, that's right, the app-strings.json was not edited. I'll correct it, sorry.